packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationMembers.tsx

```diff
@@ -53,8 +51,15 @@ export const OrganizationMembers = withCardStateProvider(() => {
           </Header.Root>
           {isAdmin && (
             <IconButton
+              aria-label='Invite'
```

Comment:
Should this be based on the localisation key?

Reply:
Yes probably. Should we tackle this in another PR along with any other similar strings that might be set?